docs/resources/cluster.md

```diff
@@ -1,100 +1,334 @@
+- `password` (String, Sensitive) Password for the user edb_admin. It must be 12 characters or more.
+- `pg_type` (String) Postgres type. e.g. "epas", "pgextended" or "postgres".
+- `pg_version` (String) Postgres version. See https://www.enterprisedb.com/docs/biganimal/latest/overview/05_database_version_policy/#supported-postgres-types-and-versions for supported Postgres types and versions.
+- `region` (String) Region to deploy the cluster. See https://www.enterprisedb.com/docs/biganimal/latest/overview/03a_region_support/ for supported regions.
+- `storage` (Block List, Min: 1) Storage. (see [below for nested schema](#nestedblock--storage))
 
 ### Optional
 
-- `allowed_ip_ranges` (Block List) Allowed IP ranges (see [below for nested schema](#nestedblock--allowed_ip_ranges))
-- `backup_retention_period` (String) Backup Retention Period.
-- `pg_config` (Block List) Instance Type (see [below for nested schema](#nestedblock--pg_config))
-- `private_networking` (Boolean) Is private networking enabled
-- `read_only_connections` (Boolean) Is read only connections enabled
+- `allowed_ip_ranges` (Block List) Allowed IP ranges. (see [below for nested schema](#nestedblock--allowed_ip_ranges))
+- `backup_retention_period` (String) Backup Retention Period. e.g. "7d", "2w" or "3m".
+- `pg_config` (Block List) Database Configuration Parameters. See https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/03_modifying_your_cluster/05_db_configuration_parameters/ for details. (see [below for nested schema](#nestedblock--pg_config))
+- `private_networking` (Boolean) Is private networking enabled.
+- `read_only_connections` (Boolean) Is read only connection enabled.
 - `timeouts` (Block, Optional) (see [below for nested schema](#nestedblock--timeouts))
 
 ### Read-Only
 
-- `cluster_id` (String) cluster ID
-- `connection_uri` (String) cluster connection uri
-- `created_at` (String) Cluster Creation Time
-- `deleted_at` (String) Cluster Deletion Time
-- `expired_at` (String) Cluster Expiry Time
-- `first_recoverability_point_at` (String) Earliest Backup recover time
+- `cluster_id` (String) Cluster ID.
+- `connection_uri` (String) Cluster connection URI.
+- `created_at` (String) Cluster Creation Time.
+- `deleted_at` (String) Cluster Deletion Time.
+- `expired_at` (String) Cluster Expiry Time.
+- `first_recoverability_point_at` (String) Earliest Backup Recover Time.
 - `id` (String) The ID of this resource.
 - `phase` (String) Current Phase of the cluster.
-- `resizing_pvc` (List of String) Resizing PVC
+- `resizing_pvc` (List of String) Resizing PVC.
+- `ro_connection_uri` (String) Cluster read-only connection URI, only available for high availability clusters.
 
 <a id="nestedblock--cluster_architecture"></a>
 ### Nested Schema for `cluster_architecture`
 
 Required:
 
-- `id` (String) ID
-- `nodes` (Number) Node Count
+- `id` (String) Cluster Architecture ID. e.g. "single", "ha" or "eha".
```

Comment:
```suggestion
- `id` (String) Cluster Architecture ID. For example, "single", "ha", or "eha".
```

Reply:
e.g. -> For example, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/4de4c956dd0831008781a5f59b9011542e378c5a
Serial comma, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/a7e9930ede32c6adc1fc0810c56a60d8391f3fb3